ppx/project.py

```diff
@@ -102,8 +103,9 @@ def local(self, path):
         if path is None:
             if config.path == Path(Path.home(), ".ppx"):
                 config.path.mkdir(exist_ok=True)
-
             self._local = Path(config.path, self.id)
```

Comment:
I need to take a look at how we could make it feasible to configure a cloud path as the ppx data directory.